### DISCUSSION TITLE
Add support for existing

### DISCUSSION THREAD

**Discussion by EchoEllet** [Luke100000/minecraft-comes-alive#854] [Q&A]
How can I replace all the villagers with male and female villagers of MCA reborn mod for an existing world? is there a way or solution around this?

**Comment by Luke100000** [comment]
```
/mca-admin convertVillagers range
```

**Reply by EchoEllet** [reply]
> ```
> /mca-admin convertVillagers range
> ``` 

convertVanillaVillagers you mean

Just found the command, thanks
